Frameworks/Logging/LoggingNative.cpp

```diff
@@ -33,10 +33,10 @@ std::mutex s_isRegisteredMutex;
 
 // TODO: WIL logging hook
 // This is where we store the WIL hook
-//namespace wil {
```

Comment:
yep.

---
In reply to: [74168773](https://github.com/Microsoft/WinObjC/pull/740#discussion_r74168773) [](ancestors = 74168773)